Vae UI update and Tooltip update

Calamdor opened the pull request:
Three Changes to the UI:
1 - Changed tooltip of "Continue from last Backup" to remove the /run from <workspace>/run/backup.
2 - Changed the VAE label to add "Read Tooltip"
3 - Changed the VAE Tooltip to add a note about diffusion format, and that using a safetensor file will cause a model load error.

Nerogar commented on modules/ui/ModelTab.py:
```diff
@@ -235,8 +235,8 @@ def __create_base_components(
-            components.label(self.scroll_frame, row, 0, "VAE",
-                             tooltip="Directory or Hugging Face repository of a VAE model. Can be used to override the base model VAE.")
+            components.label(self.scroll_frame, row, 0, "VAE (Read Tooltip)",
+                             tooltip="Directory or Hugging Face repository of a VAE model in diffusion format. Can be used to override the base model VAE. Using a safetensor VAE file will cause an error that the model cannot be loaded.")
```
The format is called diffusers, not diffusion.

Calamdor replied:
I had that initially, but google searching made me go to diffusion.  Will fix.

Nerogar commented on modules/ui/ModelTab.py:
```diff
@@ -235,8 +235,8 @@ def __create_base_components(
             # base model
-            components.label(self.scroll_frame, row, 0, "VAE",
-                             tooltip="Directory or Hugging Face repository of a VAE model. Can be used to override the base model VAE.")
+            components.label(self.scroll_frame, row, 0, "VAE (Read Tooltip)",
```
I'm not sure if "(Read Tooltip)" is a good addition to the name. Generally, all tooltips should be read. A better solution (in my opinion) would be to add a more descriptive error message when the VAE loading fails.

Calamdor replied:
It seems error reporting is done with this line at the end of a model loader? It seems difficult to tailor that to a specific error case.
raise Exception("could not load model: " + model_names.base_model)

Calamdor commented:
Changed diffusion to diffusers.
Changed VAE (Read Tooltip) to VAE Override
Updated VAE Override tooltip to explicitly mention it will override the VAE (already) included in the base model.
